CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```

Comment:
It looks like unreleased would include all the versions. is there a way to make this clearer?

Reply:
It's not included any versions; https://github.com/giantswarm/micrologger/compare/v0.3.0...HEAD

Is that the same link?

Reply:
Oh nevermind